packages/reports/src/Stores/Modules/Profit/profit-store.js

```diff
@@ -121,10 +120,11 @@ export default class ProfitTableStore extends BaseStore {
     }
 
     networkStatusChangeListener(is_online) {
-        this.is_loading = !is_online;
+        this.is_loading = this.is_loading || !is_online;
     }
 
     async onMount(shouldFilterContractTypes) {
+        const loginid = this.client_loginid;
```

Comment:
`this.client_loginid` is `undefined` on the initial load, we can rely on it to avoid unnecessary initial call of `clearTable()` below, which causes `this.is_loading` to be changed to `false`, and then immediately back to `true` inside `fetchNextBatch()`.
I've also added another `isMounting` param to `fetchNextBatch`, so that it can execute while `is_loading` is still `true` upon mounting.
CamelCase has been selected for `isMounting` to be consistent with the other existing params.